src/main/java/frc/robot/operatorInputs/Controls.java

```diff
@@ -6,7 +6,8 @@
 
```

Comment:
So what you've done here will technically work.  However, from an object oriented perspective what we want to do is revert this back how it was, and instead have two instances of Controls in Robot.  One should be driveControls, the other operatorControls, and pass them both into OperatorInputs.  Then in OperatorInputs, you would just call driverControls.getJoystickXButton() etc.  This way if we change how a button works in Controls, we only need to do it once and both controllers get it at the same time.